alphagov/notifications-admin
# Webauthn login

Review commit by commit. To test locally 

* ensure api is on latest master
* add your security key to your user profile locally
* run `update users set auth_type = 'webauthn_auth' where id = '12345';` in your local database

Things to try and test:

invite flow
email revalidation flow
signing in in separate browsers
signing in on a mobile device (may be easier to wait until preview for this one)

There's a bunch of things we're aware of that I think _don't_ need to be done in this PR, but probably should be done in this pivotal story. I've listed them as "later" tasks on the pivotal story: https://www.pivotaltracker.com/story/show/178128470
